tool/tsh/aws.go

```diff
@@ -364,6 +348,71 @@ func (a *awsApp) startLocalForwardProxy(port string) error {
 	return nil
 }
 
+// SetupKeyStore installs local CA to the specified Java KeyStore.
+func (a *awsApp) SetupKeyStore(keystore string) error {
+	aliasName := fmt.Sprintf("teleport-%v-%v", a.profile.Cluster, a.appName)
+
+	// Print info like alias name in case user wants to manually update the
+	// KeyStore. Use new lines between keytool commands as it may prompt user
+	// for password or print some outputs.
+	fmt.Fprintln(a.cf.Stdout(), "KeyStore path: ", keystore)
+	fmt.Fprintln(a.cf.Stdout(), "Local CA path: ", a.profile.AppLocalCAPath(a.appName))
+	fmt.Fprintln(a.cf.Stdout(), "Alias name   : ", aliasName)
+	fmt.Fprintln(a.cf.Stdout())
+
+	// Try a "delete" first, otherwise "import" will fail if alias already exists.
+	fmt.Fprintln(a.cf.Stdout(), "Deleting local CA from keystore.")
+	if err := a.deleteLocalCAFromKeyStore(keystore, aliasName); err != nil {
+		return trace.Wrap(err)
+	}
+
+	fmt.Fprintln(a.cf.Stdout())
+	fmt.Fprintln(a.cf.Stdout(), "Installing local CA to keystore.")
+
+	if err := a.installLocalCAToKeyStore(keystore, aliasName); err != nil {
+		return trace.Wrap(err)
+	}
+
+	fmt.Fprintln(a.cf.Stdout())
+	return nil
+}
+
+func (a *awsApp) deleteLocalCAFromKeyStore(keystore, aliasName string) error {
+	cmd := exec.Command(
+		"keytool", "-noprompt", "-delete",
```

Comment:
Could we do this without the external dependency? This looks easy to break in a variety of systems.

Reply:
`keytool` is specific to Java and is included in JDK. So if you are using AWS Java SDKs, the binary should be there. The new `--key-store` is an optional flag that tries to help users setup. But today users run `tsh proxy aws` for Java SDK v2 without this flag anyway. 

I see a few golang implementations of `keytool` like [minijks](https://github.com/lwithers/minijks) but it feels not very good.

I don't think PKCS12 will always work out either. I think `keytool` is the best bet.

Reply:
I guess one other alternative would be just printing out instructions for the users to consider and execute separately.

Reply:
I think we could consider the following:

1. Let users specify the location of keytool via an env var (default to $PATH search)
2. If we can't find keytool, we print instructions explaining the env var and/or the command they may run.

WDYT?

Reply:
Sounds good! I think we use `exec.LookPath` in some other places. We can do that first on `keytool` and if that fails we can just print the instructions.